src/components/layout/footer.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+
+import copyrightLogo from "@public/images/icons/copyright.svg";
+import githubIcon from "@public/images/icons/github.svg";
+import linkedinIcon from "@public/images/icons/linkedin.svg";
+import slackIcon from "@public/images/icons/slack.svg";
+import Image from "next/image";
+import React, { CSSProperties, useEffect, useState } from "react";
+
+const Footer = React.memo(() => {
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 768);
+    };
+
+    window.addEventListener("resize", handleResize);
+    handleResize();
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
```

Comment:
We probably don't need this logic to identify mobile screen and make responsive based on this. We can use tailwind's utilities like `sm md lg` to handle responsiveness

Reply:
Updated to use tailwind classes, let me know what you think